modules/virt-template-vmi-probe-config.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-monitoring-vm-health.adoc
+
+[id="virt-template-vmi-probe-config_{context}"]
+= Template: Virtual machine instance configuration file for defining health checks
+
+[source,yaml]
+----
+  apiVersion: kubevirt.io/v1alpha3
```

Comment:
Were you able to verify this example works or do you need help with that?

Reply:
The template is based on the existing documentation example: https://docs.openshift.com/container-platform/4.8/virt/logging_events_monitoring/virt-monitoring-vm-health.html#virt-define-readiness-probe_virt-monitoring-vm-health

But it would good to verify that it works. Can you please help with that? Thanks.